### ISSUE TITLE
What the relationship between LC-FCN and VGG-16?

### ISSUE THREAD

**Issue by MingfangDeng** [IssamLaradji/looc#2]
I confused about this paper and the code, you have two networks that produce pseudo point-level annotations and generate the localization map perspectively or one network to realize two functions ?
I really really confused. I watch the LCFCN network and it also based on VGG16, so I don't understand this problem.

**Comment by IssamLaradji** [comment]
They are two independent networks. They are both based on FCN-8 with VGG-16 architecture. The main difference is that lcfcn uses a loss function that performs counting. Check the LCFCN paper https://github.com/ElementAI/LCFCN.

**Comment by MingfangDeng** [comment]
OK, I want to know if the proposals generator is always genrator the 1000 porposals. And next circle this 1000 proposals are used in unlabeled region? And the train the LCFCN on these pseudo labels,  thsese pseudo labels  contain the pseudo labels generated in the last circle and the pseudo labels generated in this circle?  I cannot find the 1000 proposals in these code. I am thinking if the  counts of the image more than 1000, how we should do?
looking forward to your reply!

**Comment by IssamLaradji** [comment]
Yes, I am using `rcnn`'s proposal network which is in line 58 in https://github.com/IssamLaradji/looc/blob/master/src/models/attention_network.py , whose default number of output proposals is 1000
![image](https://user-images.githubusercontent.com/3382128/167017537-b56fd728-ac7c-40e0-9fe9-dae45d438337.png)

**Comment by MingfangDeng** [comment]
thank you very much!

**Comment by MingfangDeng** [comment]
I am so sorry to interrupt you. However, I want to know  what the get_tgt_list? May I understand it that is the set of location loss?
Looking forward to your reply!

**Comment by IssamLaradji** [comment]
yes it gets the set of target locations for the loss, its in line 40 in this file: https://github.com/IssamLaradji/looc/blob/50a05b9bf2d36cd8770add8cc65f9bab1ad45841/src/models/lcfcn.py#L40

**Comment by MingfangDeng** [comment]
Thank you very much! Have a good luck!
